src/tools/pub/pubspec.md

```diff
@@ -449,6 +449,24 @@ Pub.dev requires topics to follow these specifications:
 When choosing topics, consider if [existing topics]({{site.pub}}/topics)
 are relevant. Tagging with existing topics helps users discover your package.
 
+### Ignored_advisories
+
+If a package has a dependency which is affected by a security advisory, the pub
+client will warn about the advisory at dependency resolution. Package authors
+can use the 'ignored_advisories' property to allowlist an advisory that is not
+relevant for the package. To suppress the warning about an advisory, add the
+advisory identifier to the 'ignored_advisories' list. For example:
+
+```yaml
+name: myapp
+dependencies:
+  foo: ^1.0.0
+ignored_advisories:
+ - GHSA-4rgh-jx4f-qfcq
+```
+
+For more information, see [Security advisories](/tools/pub/security-advisories).
```

Comment:
We've been mostly standardizing to "check out" rather than "see":

```suggestion
For more information, check out
[Security advisories](/tools/pub/security-advisories).
```